posts/2021/10/cxx-numba-interoperability.md

```diff
@@ -0,0 +1,276 @@
+<!--
+.. title: An efficient method of calling C++ functions from numba using clang++/ctypes/rbc
+.. slug: cxx-numba-interoperability
+.. date: 2021-10-06 10:30:58 UTC-05:00
+.. author: Pearu Peterson
+.. tags: numba, c++, python, ctypes, rbc
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+
+# An efficient method of calling C++ functions from numba/Python using clang++/ctypes/rbc
+
+## Introduction
+
+The aim of this document is to explore a method of calling C++ library
+functions from [Numba](http://numba.pydata.org/) [jitted
+functions](https://numba.pydata.org/numba-doc/latest/user/jit.html#compiling-python-code-with-jit)
+--- Python functions that are decorated with
+``numpa.jit(nopython=True)``.
```

Comment:
```suggestion
``numpa.jit(nopython=True)``. Throughout this post we refer to such decorated functions as JITied or Just In Time compiled functions.
```

Reply:
With the use of "Numba compiled functions", I think this suggestion becomes unnecessary.